index.html

```diff
@@ -4975,6 +4975,634 @@ <h2 id="AudioWorkerParamDescriptor">
+              a lower bit-depth), and by quantizing in time resolution
+              (simulating a lower sample rate). This example shows how to use
+              <a><code>AudioParam</code></a>s (in this case, treated as
+              <a>a-rate</a>) inside an
+              <a><code>AudioWorkletProcessor</code></a>.
+            </p>
+            <h4>
+              Global Scope
+            </h4>
+            <pre>
+window.audioWorklet.import('bitcrusher.js').then(function () {
+  var context = new AudioContext();
+  var osc = new OscillatorNode(context);
+  var amp = new GainNode(context);
+
+  // Create a worklet node. 'BitCrusher' identifies the 
+  // AudioWorkletProcessor previously registered when
+  // bitcrusher.js was imported. The options automatically
+  // initialize the correspondingly named AudioParams.
+  var bitcrusher = new AudioWorkletNode(context, 'BitCrusher', { 
+    bitDepth: 8, 
+    frequencyReduction: 0.5
+  });
+
+  osc.connect(bitcrusher).connect(amp).connect(context.destination);
+  osc.start();
+});
+            </pre>
+            <h4>
+              AudioWorkletGlobalScope: bitcrusher.js
+            </h4>
+            <pre>
+registerAudioWorkletProcessor('BitCrusher', class extends AudioWorkletProcessor {
+
+  static get parameterDescriptors () {
+    return [{
+      name: 'bitDepth',
+      defaultValue: 12,
+      minValue: 1
+      maxValue: 16 
+    }, {
+      name: 'frequencyReduction',
+      defaultValue: 0.5,
+      minValue: 0,
+      maxValue: 1
+    }]
+  }
+
+  constructor (options) {
+    // We don't need to look at options: only AudioParams are initialized,
+    // which were taken care of by the node.
+    super(options);
+    this.phase = 0;
+    this.lastSampleValue = 0;
+  }
+
+  process (inputs, outputs, parameters) {
+    var input = inputs[0];
+    var output = outputs[0];
+    var bitDepth = parameters.bitDepth;
+    var frequencyReduction = parameters.frequencyReduction;
+
+    for (var channel = 0; channel &lt; output.length; ++channel) { 
+      for (var i = 0; i &lt; output[channel].length; ++i) {
+        var step = Math.pow(0.5, bitDepth[i]);
+        this.phase += frequencyReduction[i];
+        if (this.phase &gt;= 1.0) {
+          this.phase -= 1.0;
+          this.lastSampleValue = 
+            step * Math.floor(input[channel][i] / step + 0.5);
+        }
+        output[channel][i] = this.lastSampleValue;
+      }
+    }
+  }
+
+});
+            </pre>
+          </section>
+          <section>
+            <h3>
+              VU Meter Node
+            </h3>
+            <p>
+              This example demonstrates a simple sound level meter. It shows
+              how to exchange states (that do not require sample-accurate
+              scheduling) between <a><code>AudioWorkletNode</code></a> and
+              <a><code>AudioWorkletProcessor</code></a>. This node does not use
+              any output.
```

Comment:
It would be helpful to note here that this example further illustrates how to create an AudioWorkletNode subclass that FAPP acts like a native node, accepting constructor options and wrapping audio-thread communication in clean method calls and attribute accesses.